### DISCUSSION TITLE
Question about running DPLR MD

### DISCUSSION THREAD

**Discussion by Alexandrina-Chen** [deepmodeling/deepmd-kit#4447] [General]
Dear community,

I am recently learning how to train and run with a DPLR model. But I am facing some inconsistency between the manual (https://docs.deepmodeling.com/projects/deepmd/en/master/model/dplr.html) and a CSI workshop input (https://github.com/CSIprinceton/workshop-july-2023/blob/main/hands-on-sessions/day-3/8-deep-potential-long-range/3-md/in.lammps).

Here I quote from the manual

> ```
># compute the temperature of real atoms, excluding virtual atom contribution
>compute		real_temp real_atom temp
>compute		real_press all pressure real_temp
> fix		1 real_atom nvt temp ${TEMP} ${TEMP} ${TAU_T}
> fix_modify	1 temp real_temp
> ```
> The temperature of the system should be computed from the real atoms. The kinetic contribution in the pressure tensor is also computed from the real atoms. The thermostat is applied to only real atoms. The computed temperature and pressure of real atoms can be accessed by, e.g.

Notice that it says _"The thermostat is applied to only real atoms."_

However, in a csi workshop lammps input for DPLR

```# compute the temperature of real atoms, excluding virtual atom contribution
compute         real_temp real_atom temp
compute         real_press all pressure real_temp
#velocity        all create 330 18234589
fix             1 all npt temp 330 330  0.05 iso 1.0 1.0 0.5
fix_modify      1 temp real_temp press real_press
```

I tried to run the simulation with the above input, it seems doing well. I understand that it runs w/o error doesn't mean it is running as expected.

Therefore I just want to confirm that, is it mandatory to make sure that the `fix nvt/npt` should only be applied for the `real_atom` group?

Thank you in advance!

**Comment by njzjz** [comment]
@ChunyiZh @Yi-FanLi please take a look when you get a chance.
